scripts/benchmark.sh

```diff
@@ -30,13 +31,17 @@ function onerror {
 }
 trap onerror EXIT
 
-CLUSTER_ID="--cluster-id=0a5ca1ab1ebee11e"
-REPLICA_ADDRESSES="--replica-addresses=3001"
+CLUSTER_ID="0"
+REPLICA_ADDRESSES="--addresses=3001"
+
+echo "Initiating database file..."
+rm -f cluster_*$CLUSTER_ID_replica_*.tigerbeetle
```

Comment:
> Since the cluster is formatted `0000000000`, I will need to leave the `*` `cluster_0000000000_replica_000.tigerbeetle`

Ok, but it still needs to be:
```suggestion
rm -f cluster_*${CLUSTER_ID}_replica_*.tigerbeetle
```

because otherwise the shell will parse the variable as `$CLUSTER_ID_replica_`, which is an empty string.

Reply:
I also think it would be worth changing the `CLUSTER_ID` to `0000000000` so that we can remove that first wildcard.

Reply:
Ah!. Valid point @sentientwaffle :bow: . I updated now to `cluster_${CLUSTER_ID}_replica_*.tigerbeetle` and made `CLUSTER_ID="0000000000"`